sim/make/cpp-lint.mk

```diff
@@ -18,6 +18,8 @@ clang_tidy_files := $(shell \
 		| grep -v serial \
 		| grep -v fesvr \
 		| grep -v firesim_top \
+		| grep -v generated-src \
```

Comment:
Are you sure this is desired? If i'm understanding this correctly it might apply to generated verilator files and black box resources brought in on the classpath (amoungst others)? If so it might be wise to leave a comment explaining why we're applying these to generated files.

I guess put a different way, for my own elucidation: why `grep` and not `git grep`

Reply:
`git grep` is a good idea here.